app/views/helpers/javascript_vars.phtml

```diff
@@ -46,6 +46,7 @@ echo htmlspecialchars(json_encode([
 		'last_entry' => @$s['last_entry'],
 		'collapse_entry' => @$s['collapse_entry'],
 		'load_more' => @$s['load_more'],
+		'mylabels' => @$s['mylabels'],
```

Comment:
Note for either this PR or another PR: we should replace this syntax with something like `$s['mylabels'] ?? ''` on all these lines

Reply:
I created a ticket for it to do it outside of this PR